Release_Notes.md

```diff
@@ -0,0 +1,65 @@
+# FLEDGE Release Notes
+
+
+## Chrome M104
+
+
+
+*   Add `.well-known` join/leave permission cache.
+*   Clear pending report request queue after a certain time to prevent reports continuing for unbounded amounts of time.
+*   Implement improved process model for Android (for local FLEDGE testing by developers while the FLEDGE origin trial is not supported on Android).
+
+
+## Chrome M103
+
+
+
+*   Add `chrome://tracing` support for FLEDGE auctions (including worklets).
+*   Add basic `.well-known` support for determining if a site is allowed to join or leave interest groups for an owner.  `.well-known` ownership delegation is [documented in the explainer](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#13-permission-delegation).
+*   `joinAdInterestGroup()` and `leaveAdInterestGroup()` changed to return promises (fulfilled or rejected based on whether the join or leave is allowed based on the `.well-known` lookup).
+*   Rate-limit report network requests, to prevent over-consuming networking resources and allow requests to be sent in parallel.
+*   Add `navigator.deprecatedReplaceInURN() `function.  For more information see [Issue #286](https://github.com/WICG/turtledove/issues/286).
+*   Bug fixes:
+    *   [Fix error when component auction does not bid](https://bugs.chromium.org/p/chromium/issues/detail?id=1321941).
+
+
+## Chrome M102
+
+
+
+*   [Fenced Frame reporting](https://github.com/WICG/turtledove/blob/main/Fenced_Frames_Ads_Reporting.md) added. Worklets now support a `registerAdBeacon()`  function in `reportWin()` and `reportResult()` which can be used to register reporting URLs.
+*   As described in the FLEDGE explainer, ads selected by a FLEDGE `runAdAuction()` call and rendered in a Fenced Frame can now [call navigator.leaveAdInterestGroup() without arguments](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#13-permission-delegation:~:text=invoking%20navigator.leaveAdInterestGroup()-,from%20inside%20an%20ad,-that%20is%20being) to leave the interest group that triggered the ad.
+*   Limits on the number of interest groups allowed to participate in auctions cause auctions to choose interest groups uniformly and randomly they they have equal priority instead of in order of decreasing expiration time. See `perBuyerGroupLimits` [in the FLEDGE explainer](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#12-interest-group-attributes) for more information about interest group limits in auctions.
```

Comment:
```suggestion
*   Limits on the number of interest groups allowed to participate in auctions cause auctions to choose interest groups uniformly and randomly. They have equal priority instead of in order of decreasing expiration time. See `perBuyerGroupLimits` [in the FLEDGE explainer](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#12-interest-group-attributes) for more information about interest group limits in auctions.
```